dhcpv4/nclient4/conn_darwin.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2018 the u-root Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.12
```

Comment:
I feel like a build tag on the other file instead of copy & pasting it would be preferred? 

https://github.com/insomniacslk/dhcp/pull/457